W-11673772 upcomingprocesschanges se (#176)
mulesoft/docs-rpa: rpa-builder/6.2/modules/ROOT/pages/activity-library.adoc

+
This method creates a local copy of the workflow behind the activity and inserts it into the project, which enables you to make changes. However, because there is no reference to the activity in the Activity Library, the local copy does not automatically update if the activity in the Activity Library changes. 

[[change-the-revision-of-a-referenced-activity]]
== Change the Revision of a Referenced Activity

> If the latest revision of a referenced activity does not work properly in a process or if you accidentally xref:rpa-manager::processoperations-upcomingprocesschanges.adoc#deploy-the-latest-version-of-an-activity[deployed the latest revision using RPA Manager], you can revert the activity to another revision:

Reviewer: ```suggestion
If the latest revision of a referenced activity does not work properly in a process or if you accidentally xref:rpa-manager::processoperations-upcomingprocesschanges.adoc#deploy-the-latest-version-of-an-activity[deployed the latest revision using RPA Manager], follow these steps to revert the activity to an earlier revision:
```

Author: You can also revert the activity to a later revision again.

Author: --> replaced with: to an earlier or later revision

Author: If the deployed revision of a referenced activity does not work properly in a process or if you accidentally xref:rpa-manager::processoperations-upcomingprocesschanges.adoc#deploy-the-latest-version-of-an-activity[deployed the latest revision using RPA Manager], follow these steps to revert the activity to an earlier or later revision:
